LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2025 Radio Cosmology
+Copyright (c) 2025 CHIME collaboration 
```

Comment:
I realise we do this all over the place, but this isn't really valid.  The CHIME Collaboration isn't a legal entity for this purpose.  It cannot be the owner of a copyright in Canada.

I would replace this with one or more names.

Reply:
I'd suggest Seth and Tristan, who I think created most of the catalogues.